### PR TITLE
Fix index.md links

### PR DESCRIPTION
**Pull request by alanpoulain**

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -99,21 +103,13 @@ exports.onCreateNode = ({ node, getNode, actions }) => {
...
     localUrls.map(url => {
-      let newUrl = url.replace('.md', '/');
+      let newUrl = url.replace(/(\/index)?\.md/, '/');
       newUrl = `/${URL.resolve(nodePath, newUrl)}`;
       html = html.replace(url, newUrl);
       return true;
     });
 
     node.internal.content = html;
-    if ('index' === path.basename(nodePath)) {
```
Comment (dunglas) [inline gatsby-node.js]: Could you keep that for now please? IIRC it's because we used to have such links, so it's to prevent a 404 for old links.

Comment (alanpoulain) [reply gatsby-node.js]: It's not working. The code for managing these nodes has been removed. Maybe we can redo it in another PR?

Comment (alanpoulain) [reply gatsby-node.js]: The code was this: https://github.com/api-platform/website/blob/f0a81f24a4bdfcf1422b4c3887463114371a61f1/gatsby-node.js#L129-L141

**Comment by alanpoulain** [comment]
@dunglas I've added the redirections.

**Comment by teohhanhui** [comment]
There are no tests that can prevent regressions like this in the future?

**Comment by alanpoulain** [comment]
There is no test in this project, sadly. We could add Cypress or something like that.

**Pull request merged**